src/config/default.tsx

```diff
@@ -0,0 +1,16 @@
+export default {
```

Comment:
Should this be in the internal-infra repo?

Reply:
We can move it internal infra, but I guess at minimum we would have to leave a copy of this file in place with some blank values to help folks who want to run their own instances. @vishakh thoughts?

Reply:
It's fine to have here since it only contains public URLs.